src/routes/lib/v1/PackageRequest.js

```diff
@@ -125,6 +127,25 @@ class PackageRequest extends BaseRequest {
 		return listing;
 	}
 
+	async getEntrypoints () {
+		let props = { type: this.params.type, name: this.params.name, version: this.params.version };
+		let packageEntrypoints = await PackageEntrypoints.find(props);
+
+		if (packageEntrypoints) {
+			return JSON.parse(packageEntrypoints.entrypoints);
+		}
+
+		let { entrypoints } = await jsDelivrRemoteService.usingContext(this.ctx).listResolvedEntries(this.params.type, this.params.name, this.params.version);
+
+		if (!entrypoints) {
+			return [];
+		}
+
+		await new PackageEntrypoints({ ...props, entrypoints: JSON.stringify(entrypoints) }).insert().catch(() => {});
```

Comment:
There's actually no need for `await`  when we ignore errors anyway, you can change it in `getFilesAsJson()` too.

Reply:
This still applies.

Reply:
fixed